blockchain/chain.go

```diff
@@ -834,6 +834,24 @@ func (b *BlockChain) calcSequenceLock(tx *btcutil.Tx, utxoView *UtxoViewpoint,
 	return sequenceLock, nil
 }
 
+// LockTimeToSequence converts the passed relative locktime to a sequence
+// number in accordance to BIP-68.
+// See: https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki
+//  * (Compatibility)
+func LockTimeToSequence(isSeconds bool, locktime uint32) uint32 {
+	// If we're expressing the relative lock time in blocks, then the
+	// corresponding sequence number if simply the desired input age.
```

Comment:
Hmm it's possibly the most recent rebase got a bit hairy, just pushed a commit fixing it (again).